src/cli/data.py

```diff
@@ -566,6 +566,14 @@ def new(self):
                     "help": "IDs of the resources to delete",
                 },
             ),
+            (
+                ["--database"],
+                {
+                    "type": str,
+                    "required": False,
+                    "help": "IDs of database that columns are in",
```

Comment:
Should this be plural. as in:

> IDs of the databases that the columns are in

Reply:
it is. we can delete many columns in 1 DB, but they all have to be in one DB

Reply:
also, it *has* to be plural to delete other objects

Reply:
Since its one database (singular), I think "ID" singular would be clearer.

Reply:
oof, you're right. my bad. got confused with IDs of objects

Reply:
> also, it has to be plural to delete other objects

In that case, should it be this:

> IDs of the databases that the columns are in

Reply:
fixed

Reply:
no, you're right. it's ID singular for DB, and IDs plural for objects